include/ctranslate2/storage_view.h

```diff
@@ -3,6 +3,11 @@
 #include <ostream>
 #include <vector>
 
+#ifdef _MSC_VER
+#  include <BaseTsd.h>
+   typedef SSIZE_T ssize_t;
```

Comment:
That's OK for now. In the future, we probably want to use revisit the types used for dimension values. I think we could use `int64_t` everywhere.